modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -253,6 +260,35 @@ namespace imgproc {
 
 }
 
+// TODO: move to the separate file modules/gapi/include/opencv2/gapi/video.hpp
+namespace  video {
+    using GPoint2fArray        = cv::GArray<cv::Point2f>;
+    using GUcharArray          = cv::GArray<uchar>;
+    using GFloatArray          = cv::GArray<float>;
+    using GGMatArray           = cv::GArray<cv::GMat>;
+
+    using GOptFlowLKOutput     = std::tuple<GPoint2fArray, GUcharArray, GFloatArray>;
```

Comment:
Can out G_TYPED_KERNEL macro work without this?

Reply:
What do you mean? There shouldn't be this _using_?

Reply:
Leaved as it was to reduse the use of long tuples in some places